### PR TITLE
Release notes updated to reflect Cloud controller manager for Alibaba Cloud as TP

### PR DESCRIPTION
**Pull request by ahardin-rh**
Preview build: https://deploy-preview-41887--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes.html#ocp-4-10-notable-technical-changes

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -373,9 +373,9 @@ Users also have the option to use the `REGISTRY_AUTH_FILE` environment variable,
...
+The Kubernetes community plans to deprecate the Kubernetes controller manager in favor of using cloud controller managers to interact with underlying cloud platforms. As a result, there is no plan to add Kubernetes controller manager support for any new cloud platforms. The IBM Cloud implementation that is added in this release of {product-title} uses cloud controller managers.
 
-In addition, this release supports using cloud controller managers for Google Cloud Platform (GCP) and VMware vSphere as a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview].
+In addition, this release supports using cloud controller managers for Google Cloud Platform (GCP), VMware vSphere, and Alibaba Cloud as a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview].
```
Comment (mburke5678) [inline release_notes/ocp-4-10-release-notes.adoc:378]: Perhaps this is my ignorance. This wording _this release supports using cloud controller managers for_ makes it sound like the CCM is an optional feature I can use. Is that right?

**Comment by mburke5678** [comment]
@ahardin-rh One pesky question. Otherwise LGTM.

**Comment by mburke5678** [comment]
@ahardin-rh We got a [request in Slack](https://coreos.slack.com/archives/C01PES793LL/p1645455382370369) from @gauravsingh85 to add some information to the Alibaba release notes: _two region in internaltional portal does not support IPI [ cn-nanjing (China (Nanjing) ,UAE (Dubai) ]_  Is this something you can help us with?

**Comment by gauravsingh85** [comment]
Looks good to me

**Comment by ahardin-rh** [comment]
@mburke5678  Yes, I will open a separate PR to cover those details. Thanks!
@gauravsingh85 Thank you!

**Pull request merged**